perun-openapi/openapi.yml

```
@@ -7193,6 +7193,21 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/usersManager/getAssignedRichResources:
```

```suggestion
  /json/usersManager/getAssignedRichResources:
```
When using the `GET` method, you can use the `json` (de)serializer